utils/traits.py

```diff
@@ -245,7 +248,7 @@ def reshape_1D_kernel(self, kernel):
 
     @staticmethod
     def n_joints(entity):
-        return [len(parents) for parents in entity.parents_list] #[1, 2, 6, 10, 15]
+        return [len(parents) for parents in entity.parents_list]  # TODO: just return entity.n_edges
```

Comment:
class Joint does not have a member n_edges. The comment you deleted is an example of a possible return value. Please undelete it. You may add the text "Example" in its prefix for clarity.